### PR TITLE
[eventhubs] Add more logs and retries

### PR DESCRIPTION
**Pull request by iskakaushik**

### CODE REVIEW

File: flow/connectors/eventhub/eventhub.go [hunk 1]
```diff
@@ -203,10 +199,6 @@ func (c *EventHubConnector) SyncRecords(req *model.SyncRecordsRequest) (*model.S
 		shutdown <- true
 	}()
 
-	eventsPerBatch := int(req.PushBatchSize)
```
Comment (serprex) [inline flow/connectors/eventhub/eventhub.go:206]: Should `PushBatchSize` be marked deprecated?

Comment (iskakaushik) [reply flow/connectors/eventhub/eventhub.go:206]: yup, will do!

Comment (iskakaushik) [reply flow/connectors/eventhub/eventhub.go:206]: but only for eventhub

**Comment by iskakaushik** [comment]
landing for customer mirage

**Pull request merged**